LaraApi/src-lara/lara/Tool.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+class Tool {
```

Comment:
Classes in the package 'lara' are all utility classes with static methods, this class would break that convention. Consider moving to the package 'lara.tool'. Related class ToolUtils could also go to that package.

Reply:
Done